tests/run-make/link-framework/rmake.rs

```diff
@@ -0,0 +1,29 @@
+// Check that linking to a framework actually makes it to the linker.
+
+//@ only-apple
+
+use run_make_support::{cmd, rustc};
+
+fn main() {
+    rustc().input("dep-link-framework.rs").run();
+    rustc().input("dep-link-weak-framework.rs").run();
+
+    rustc().input("empty.rs").run();
+    cmd("otool").arg("-L").arg("no-link").run().assert_stdout_not_contains("CoreFoundation");
+
+    rustc().input("link-framework.rs").run();
+    let out = cmd("otool").arg("-L").arg("link-framework").run();
```

Comment:
The API is fluent, so you can also write it like this:
```rust
cmd("otool").arg("-L").arg("link-framework").run()
    .assert...()
    .assert...();
```
(just saying, this is also fine).